web/src/app/actions/alerts.js

```diff
@@ -22,6 +22,11 @@ export function setAlertsAllServicesFilter(bool) {
   return setURLParam('allServices', bool ? '1' : null)
 }
 
+// setAlertsAllServicesFilter will set the alert list to include all services.
+export function setAlertsShowAsTimestampsFilter(bool) {
+  return setURLParam('timestamps', bool ? '1' : null)
```

Comment:
maybe `fullTime` or `absTime`? timestamps is a bit ambiguous

Reply:
Sounds good, I'll do fullTime (or whatever `DATETIME` format we land on) as the URL param